src/net/node/NetworkNode.ts

```diff
@@ -72,7 +72,9 @@ export class NetworkNode extends Network {
     getIpMac(netInterface: string): { mac: string; ips: string[]; } | undefined {
         const netInterfaceInfo = networkInterfaces()[netInterface];
         if (netInterfaceInfo === undefined) return undefined;
-        return { mac: netInterfaceInfo[0].mac, ips: netInterfaceInfo.map(({address}) => address) };
+        // only use local IPv6 address
+        const ips = netInterfaceInfo.map(({address}) => address).filter(ip => isLocalIPv6Address(ip))
```

Comment:
2.5.6.4. IPv4 Coexistence
Matter devices SHALL be tolerant of IPv4 addresses and MAY ignore those addresses for the pur­poses of Matter operations.

I interpret this as there is NO need to advertise on the IPv4 network as Matter requires an IPv6 network to function.

Reply:
In hindsite - It might be useful to provide an error upon using the library on a network interface without IPv6.

Reply:
I agree, according to specs we could kill ipv4 in announcement. But honestly it also do not hurt to still announce there.

Reply:
Previously when advertising the global IPv6 address as well as the ULA, the Home App was unable to 'find' the virtual device to commission when using a Linux instance.  Now that node-matter only advertises the ULA as defined in the spec, Home App had no issue commissioning.  I am going to strongly recommend we NOT advertise the global IPv6 address and adhere to the spec.

Reply:
I will discuss this with some more devs tonight. I can only say that for me all works with Apple, Google and Alexa, also when they use the global adress because it seesm to be correctly routed for me ... So the interesting question is why not for you?

Reply:
can you provide an example of a 'global' address on your system?

Reply:
Sure, here two network interfaces as ifconfig shows them

```
bridge0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	options=3<RXCSUM,TXCSUM>
	ether c8:2a:14:59:a1:8e 
	inet6 fe80::83a:a16e:cdc2:bbd5%bridge0 prefixlen 64 secured scopeid 0xd 
	inet 192.168.178.57 netmask 0xffffff00 broadcast 192.168.178.255
	inet6 2a02:8071:2c87:399c:1cc5:8756:a18d:3ecf prefixlen 64 autoconf secured 
	inet6 2a02:8071:2c87:399c:d90a:8773:6b1c:915a prefixlen 64 autoconf temporary 
	Configuration:
		id 0:0:0:0:0:0 priority 0 hellotime 0 fwddelay 0
		maxage 0 holdcnt 0 proto stp maxaddr 100 timeout 1200
		root id 0:0:0:0:0:0 priority 0 ifcost 0 port 0
		ipfilter disabled flags 0x0
	member: en11 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 4 priority 0 path cost 0
	member: en1 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 11 priority 0 path cost 0
	member: en2 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 10 priority 0 path cost 0
	member: en3 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 9 priority 0 path cost 0
	member: en4 flags=3<LEARNING,DISCOVER>
	        ifmaxaddr 0 port 12 priority 0 path cost 0
	nd6 options=201<PERFORMNUD,DAD>
	media: autoselect
	status: active

en0: flags=8863<UP,BROADCAST,SMART,RUNNING,SIMPLEX,MULTICAST> mtu 1500
	options=6463<RXCSUM,TXCSUM,TSO4,TSO6,CHANNEL_IO,PARTIAL_CSUM,ZEROINVERT_CSUM>
	ether 88:66:5a:44:3c:4c 
	inet6 fe80::142e:c5db:53f:cd10%en0 prefixlen 64 secured scopeid 0x7 
	inet 192.168.178.159 netmask 0xffffff00 broadcast 192.168.178.255
	inet6 2a02:8071:2c87:399c:8a0:8914:4763:faad prefixlen 64 autoconf secured 
	inet6 2a02:8071:2c87:399c:cdfc:1c49:5275:8b9c prefixlen 64 autoconf temporary 
	nd6 options=201<PERFORMNUD,DAD>
	media: autoselect
	status: active

```